Source/control.cpp

```diff
@@ -563,6 +561,67 @@ void RemoveGold(int pnum, int goldIndex)
 	dropGoldValue = 0;
 }
 
+bool GetSpellListSelection(spell_id &pSpell, spell_type &pSplType)
+{
+	pSpell = spell_id::SPL_INVALID;
+	pSplType = spell_type::RSPLTYPE_INVALID;
+
+	uint64_t mask;
+
+	Point location;
+	int &x = location.x;
+	int &y = location.y;
+	x = PANEL_X + 12 + SPLICONLENGTH * SPLROWICONLS;
+	y = PANEL_Y - 17;

```

Comment:
Why use a `Point`, if the `Point` is never used?
```suggestion
	int x = PANEL_X + 12 + SPLICONLENGTH * SPLROWICONLS;
	int y = PANEL_Y - 17;
```

Reply:
I simply like `Point` struct 😁 
Fixed